### DISCUSSION TITLE
Change value of work time entry after a period has been approved

### DISCUSSION THREAD

**Discussion by klausjettkant** [kimai/Paid-plugins#129] [Questions & Answers] [answered]
When I am billing my costumers, I use exports to prove the delivery of accomplished activities. All the work that has been done is written down in the description field of a work time entry. I want to correct a spelling error of one of my employees, before the export is sent to my customer, but I get the error message: "This period has already been approved, you cannot edit or create new entries before 01.01.24." How can i edit a description entry when it is this far in the past? Thank you in advance.

**Comment by kevinpapst** [comment, marked as answer]
Right now, there is no way to overwrite this check.
In emergency cases, you can edit it via database `kimai2_timesheets` table, but that is not really a feasible way.
Not sure what to do here.

**Reply by christian-lo** [reply]
hi,
thats an interesting use-case & might affect myself someday. would it be an option to include this into the user-roles (e.g. allow to change locked timesheets) or as part of the settings (e.g. allow edit of description of locked timesheets).
cheers, christian

**Reply by kevinpapst** [reply]
I think the description is not relevant for bookkeeping, so it it shouldn't cause an issue to edit it, even if the timesheet is locked. But I am strictly against changing date/time/duration.

The existing "edit exported timesheet" permissions could be re-used. 
Or the permission to allow lockdown overwrites.

The only issue is to build a new form that allows only description (and maybe tags) editing.

**Reply by klausjettkant** [reply]
thanks for your quick reply. i can live with that matter and the database edit works for me. if you have a feature-wishlisht list, i would be happy if you someday may find a way to let us change "description", "billable" and "project" on entries in approved years. you are doing great work, thank you!

**Reply by dl6nm** [reply]
I am running in a similar problem, where I have to change the activity of some time entries to other ones, because it was the false one selected. The database trick doesn't solve my problem, because of I am paying for a Kimai-Cloud subscription and have no access to the tables.

So what is the way to go in my case?

**Reply by kevinpapst** [reply]
Please use the Cloud support channel.
